src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/in_0.py

```diff
@@ -0,0 +1,5 @@
+for i in range(100):
```

Comment:
shouldn't this file be in a 'for' subdirectory?

Reply:
It is `break` and `continue` test case...It is general for both loops. But I can do it for both loops if you want